charts/stable/unifi/templates/syslog-svc.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if not .Values.unifiedService.enabled }}
+{{ if .Values.syslogService.enabled }}
```

Comment:
```suggestion
{{ if (not .Values.unifiedService.enabled) and .Values.syslogService.enabled }}
```

Reply:
I think this makes the behavior more explicit, and won't allow two services for the same thing. Maybe it would be best to apply this style of logic to all the non-unified services.

Reply:
Syslog is used for remote logging and not necessary.
I also see unifiedService is dependent on Syslog, speedtestService, etc., in unified-svc.yaml.
In my opinion, the unified services should separate necessary services and optionally services.